tests/examples/contracts/x_username_storage.py

```
@@ -0,0 +1,56 @@
+    username: str
+    tweet_api_url: str
+
+    def __init__(self):
+        self.tweet_api_url = "https://domain.com/api/twitter"
+        self.username = ""
+
+    @gl.public.view
+    def get_username(self) -> str:
+        return self.username
+
+    @gl.public.write
```

Ruff (0.12.2)

`gl` may be undefined, or defined from star imports

(F405)